vue-frontend/src/components/jobs/FaqSection.vue

```diff
@@ -22,7 +22,7 @@
         />
       </div>
       <div class="tw-flex-[67%]">
-        <div class="faq-content tw-pl-[60px]">
+        <div class="faq-content sm:tw-pl-[60px]">
```

Comment:
I think we need pl for large devices, in small devices it should be centered, no padding is there